views/main.html

```diff
@@ -5,7 +5,8 @@
       <div list list-id="main-content" list-prefix="<div>" list-suffix="</div>"></div>
       <map></map>
     </div>
-    <timeline-panel ng-if="mainCtrl.timeline"></timeline-panel>
+    <timeline-panel ng-if="mainCtrl.timeline"></timeline-panel><!-- TODO move into dock-bottom? -->
```

Comment:
is there a ticket for that todo?

Reply:
I started looking at putting the timeline into the new "docked window" specifically in the dock-bottom area, but it looks like it'd take some enhancements (resize and some of the settings pass-through's) to be able to support it. I can either remove the comment or leave it around as a reminder for when/if those features get added.

Reply:
you can leave it.